build.gradle

```diff
@@ -8,9 +8,9 @@ buildscript {
 
     ext {
         opensearch_group = "org.opensearch"
-        opensearch_version = System.getProperty("opensearch.version", "1.3.9-SNAPSHOT")
+        opensearch_version = System.getProperty("opensearch.version", "1.3.10-SNAPSHOT")
         // 1.3.0 -> 1.3.0.0, and 1.3.0-SNAPSHOT -> 1.3.0.0-SNAPSHOT
-        opensearch_build = opensearch_version.replaceAll(/(\.\d)([^\d]*)$/, '$1.0$2')
+        opensearch_build = opensearch_version.tokenize('-')[0] + '.0'
```

Comment:
Can you try with `opensearch_build = opensearch_version.replaceAll(/(\.\d+)([^\d]*)$/, '$1.0$2')`